demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1170,7 +1172,7 @@ def get_id_set_file(skip_dependencies, id_set_path):
         """
         id_set = {}
         if not os.path.isfile(id_set_path):
-            if not skip_dependencies:
+            if not skip_id_set_creation:
```

Comment:
1. Let's make this one condition
2. Still not seeing why this is needed - when do we not want to create an ID set, given no path to an existing ID set has been specified?

Reply:
@guykeller - some users do not need the id_set validation also this is in use in some test cases. This is an extra feature for this command so the capability to turn it off is a good option.

Reply:
So let's add that to the PR, so people will be able to know why it was created